packages/main/src/system/windows-startup.ts

```diff
@@ -55,6 +55,16 @@ export class WindowsStartup {
     const preferencesConfig = this.configurationRegistry.getConfiguration('preferences');
     const minimize = preferencesConfig.get<boolean>('login.minimize');
 
+    // after https://github.com/podman-desktop/podman-desktop/pull/13056 there is a leftover startup file that we need to remove
+    const windowsStartupFoler = path.resolve(app.getPath('appData'), 'Microsoft/Windows/Start Menu/Programs/Startup');
```

Comment:
```suggestion
    const windowsStartupFoler = path.resolve(app.getPath('appData'), 'Microsoft', 'Windows', 'Start Menu', 'Programs', 'Startup');
```

I think this might be better than using manual delimiter

Reply:
Here I've just reverted previously removed lines